x/oracle/keeper/keeper.go

```diff
@@ -28,6 +31,23 @@ func NewKeeper(
 	}
 }
 
+// GetPort returns the portID for the IBC app module. Used in ExportGenesis
+func (k Keeper) GetPort(ctx sdk.Context) string {
+	store := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	return string(store.Get(types.PortKey))
+}
+
+// SetPort sets the portID for the IBC app module. Used in InitGenesis
+func (k Keeper) SetPort(ctx sdk.Context, portID string) {
+	store := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	store.Set(types.PortKey, []byte(portID))
+}
+
+func (k Keeper) DeletePort(ctx sdk.Context) {
```

Comment:
remove

Reply:
Can't remove, used during migration

Reply:
I can't find this function in main branch, so it was already removed